docs/api-reference/layer.md

```diff
@@ -235,6 +235,39 @@ Used to override the default shallow comparison of the `data` object.
 
 As an illustration, the app could set this to e.g. 'lodash.isequal', enabling deep comparison of the data structure. This particular examples would obviously have considerable performance impact and should only be used as a temporary solution for small data sets until the application can be refactored to avoid the need.
 
+##### `_dataDiff` (Function, optional) **Experimental**
+
+This function is called when the data changes, either shallowly when `dataComparator` is not supplied or because `dataComparator` returns `false`, to retrieve the indices of the changed objects. By default, when data changes, the attributes of all objects are recalculated. If this prop is supplied, only the attributes of the specified objects will be updated. This can lead to significant performance improvement if a few rows in a large data table need to change frequently:
```

Comment:
Nit:  `the attributes of all objects are recalculated` to `the attributes of all instances are recalculated` ?  Or clarify `object` is a rendering instance.

Reply:
We do not have the concept "instance" in our documentation. From the user's perspective, an "object" is an item in the data array.